test/unit/test_mail_quoting.js

```diff
@@ -393,8 +415,9 @@ commonCase('Quoting', function(T) {
         eCheck.expect(tdef.chunks[i], tdef.chunks[i+1]);
       }
       var roundtrip = true;
-      if (tdef.hasOwnProperty('roundtrip'))
+      if ('roundtrip' in tdef) {
```

Comment:
Remembering a little more, I think I might have gotten a weird error and maybe suspected that I'd been doing Object.create(null) cleverness that broke hasOwnProperty as invoked off of `tdef` and figured "in" was prettier than using Object directly.  I think I might have added some wrapper transform function in the list previously that screwed up and forgot to return things...  Obviously not used or needed now.